specification/devcenter/DevCenter/main.tsp

```diff
@@ -1,6 +1,7 @@
 import "@typespec/rest";
 import "@typespec/http";
 import "@azure-tools/typespec-azure-core";
+import "@typespec/versioning";
```

Comment:
do we need to add this package here?

Reply:
Huh you're right, this isn't needed. Do you know why I'm still able to do `using TypeSpec.Versioning;` without this import?

Reply:
It's part of the azure core package